src/vcpkg/commands.env.cpp

```diff
@@ -23,28 +23,49 @@ namespace
     constexpr StringLiteral OPTION_PYTHON = "python";
 
     constexpr CommandSwitch SWITCHES[] = {
-        {OPTION_BIN, []() { return msg::format(msgCmdEnvOptions, msg::path = "bin/", msg::env_var = "PATH"); }},
+        {OPTION_BIN,
+         [] {
+             return msg::format(
+                 msgCmdEnvOptions, msg::path = "bin/", msg::env_var = format_environment_variable("PATH"));
+         }},
         {OPTION_INCLUDE,
-         []() { return msg::format(msgCmdEnvOptions, msg::path = "include/", msg::env_var = "INCLUDE"); }},
+         [] {
+             return msg::format(
+                 msgCmdEnvOptions, msg::path = "include/", msg::env_var = format_environment_variable("INCLUDE"));
+         }},
         {OPTION_DEBUG_BIN,
-         []() { return msg::format(msgCmdEnvOptions, msg::path = "debug/bin/", msg::env_var = "PATH"); }},
-        {OPTION_TOOLS, []() { return msg::format(msgCmdEnvOptions, msg::path = "tools/*/", msg::env_var = "PATH"); }},
+         [] {
+             return msg::format(
+                 msgCmdEnvOptions, msg::path = "debug/bin/", msg::env_var = format_environment_variable("PATH"));
+         }},
+        {OPTION_TOOLS,
+         [] {
+             return msg::format(
+                 msgCmdEnvOptions, msg::path = "tools/*/", msg::env_var = format_environment_variable("PATH"));
+         }},
         {OPTION_PYTHON,
-         []() { return msg::format(msgCmdEnvOptions, msg::path = "python/", msg::env_var = "PYTHONPATH"); }},
+         [] {
+             return msg::format(
+                 msgCmdEnvOptions, msg::path = "python/", msg::env_var = format_environment_variable("PYTHONPATH"));
+         }},
     };
 
 } // unnamed namespace
 
 namespace vcpkg
 {
-    constexpr CommandMetadata CommandEnvMetadata = {
-        [] {
-            return create_example_string(
-                fmt::format("env <{}> --triplet x64-windows", msg::format(msgOptionalCommand)));
+    constexpr CommandMetadata CommandEnvMetadata{
+        "env",
+        msgHelpEnvCommand,
+        {
+            "vcpkg env --triplet x64-windows",
+            msgCommandEnvExample2,
+            "vcpkg env \"ninja -C build\" --triplet x64-windows",
```

Comment:
```suggestion
            "vcpkg env \"ninja --version\" --triplet x64-windows",
```
A relative path argument to ninja is a bit fraught here since env doesn't retain the outer CWD.